index.html

```diff
@@ -0,0 +1,120 @@
+        
+        To date, Chris has written hundreds of articles on the subject of productivity, and has garnered coverage in media as diverse as The New York Times, The Wall Street Journal, New York magazine, The Huffington Post, Harvard Business Review, TED, Fortune, Fast Company, and Lifehacker. This talk was given at a TEDx event using the TED conference format but independently organized by a local community. Learn more at https://www.ted.com/tedx
+    </h5> 
+    <div >
+        <button type="button" id="like_button"> </button>  
+        <button type="button" id="dislike_button"> </button>
+        <button type="button" id="share_button"> </button>
+        <button type="button" id="save_button"> </button>
+    </div>
+    <div>
+        <div class="comment comment_block">
+           <img src="img/C1.jpg" alt="image" class="C1">
+           <h5 class="commentLine1">I've learnt so much from youtube. Without it I wouldnt be who I am now, but at the same time I know it distracts me from my goals and causes procrastination. Its such a double edged sword.</h5>
+        </div>
+        <div class="comment comment_block">
+            <img src="img/C2.jpg" alt="image" class="C1">
+            <h5 class="commentLine1">I'm an English student, and this guy speaks so clearly that I can understand pretty much everything, thank you so much for the content!</h5>
+         </div>
+         <div class="comment comment_block">
+            <img src="img/C3.jpg" alt="image" class="C1">
+            <h5 class="commentLine1">1. do your homework now, YES NOW
+                2. make yourself bored for a month (by getting rid of distracting things and entertainment like cellphone)   </h5>
+         </div>
+         <div class="comment comment_block">
+            <img src="img/C4.jpg" alt="image" class="C1">
+            <h5 class="commentLine1">Oh my dear this TED Talk changed my life for ever! I'm dealing with my focus since years and nobody could help me to improve it. This guy did! Thank you so much!!!</h5>
+         </div>
+         <div class="comment comment_block">
+            <img src="img/C5.jpg" alt="image" class="C1">
+            <h5 class="commentLine1">I have been off social media for 38 days and have came up with more life realizations and business/ life plans that I ever thought possible. I can also read for hours upon hours on end getting so dived up in what im reading that I forget the time. My mind simply has no reason to get distracted because there is nothing to distract my mind. This stuff works, where attention goes, energy flows! goodluck my friends</h5>
+         </div>
+    </div>
```

Comment:
please use ul tag instead of div at line 49 as <ul> is more semantic and don't forget to replace all the child divs inside to <li> tag